cl/search/models.py

```diff
@@ -3075,155 +2891,26 @@ def ordered_opinions(self):
-
-    def as_search_list(self):
-        # IDs
-        out = {}
-
-        # Court
-        court = {
-            "court_id": self.docket.court.pk,
-            "court": self.docket.court.full_name,
-            "court_citation_string": self.docket.court.citation_string,
-            "court_exact": self.docket.court_id,
-        }
-        out.update(court)
-
-        # Docket
-        docket = {
-            "docket_id": self.docket_id,
-            "docketNumber": self.docket.docket_number,
-        }
-        if self.docket.date_argued is not None:
-            docket["dateArgued"] = midnight_pt(self.docket.date_argued)
-        if self.docket.date_reargued is not None:
-            docket["dateReargued"] = midnight_pt(self.docket.date_reargued)
-        if self.docket.date_reargument_denied is not None:
-            docket["dateReargumentDenied"] = midnight_pt(
-                self.docket.date_reargument_denied
-            )
-        out.update(docket)
-
-        # Cluster
-        out.update(
-            {
-                "cluster_id": self.pk,
-                "caseName": best_case_name(self),
-                "caseNameShort": self.case_name_short,
-                "panel_ids": [judge.pk for judge in self.panel.all()],
-                "non_participating_judge_ids": [
-                    judge.pk for judge in self.non_participating_judges.all()
-                ],
-                "judge": self.judges,
-                "citation": [str(cite) for cite in self.citations.all()],
-                "scdb_id": self.scdb_id,
-                "source": self.source,
-                "attorney": self.attorneys,
-                "suitNature": self.nature_of_suit,
-                "citeCount": self.citation_count,
-                "status": self.get_precedential_status_display(),
-                "status_exact": self.get_precedential_status_display(),
-                "sibling_ids": [
-                    sibling.pk for sibling in self.sub_opinions.all()
-                ],
-            }
-        )
-        try:
-            out["lexisCite"] = str(
-                self.citations.filter(type=Citation.LEXIS)[0]
-            )
-        except IndexError:
-            pass
-        try:
-            out["neutralCite"] = str(
-                self.citations.filter(type=Citation.NEUTRAL)[0]
-            )
-        except IndexError:
-            pass
-
-        if self.date_filed is not None:
-            out["dateFiled"] = midnight_pt(self.date_filed)
-        try:
-            out["absolute_url"] = self.get_absolute_url()
-        except NoReverseMatch:
-            raise InvalidDocumentError(
-                "Unable to save to index due to missing absolute_url "
-                "(court_id: %s, item.pk: %s). Might the court have in_use set "
-                "to False?" % (self.docket.court_id, self.pk)
-            )
-
-        # Opinion
-        search_list = []
-        text_template = loader.get_template("indexes/opinion_text.txt")
```

Comment:
I think everything in the cl.search.templates.indexes directory can be removed, right?

Reply:
That's correct. I'll remove the directory